File: check_schema.py
import os, sys
from supabase import create_client
from dotenv import load_dotenv
load_dotenv()
sb = create_client(os.environ['SUPABASE_URL'], os.environ['SUPABASE_SERVICE_ROLE_KEY'])
res = sb.table('league_config').select('*').limit(1).execute()
print(res.data)

File: check_schema.js
require('dotenv').config({ path: './frontend-web/.env.local' });
const { createClient } = require('@supabase/supabase-js');

const supabase = createClient(process.env.NEXT_PUBLIC_SUPABASE_URL, process.env.SUPABASE_SERVICE_ROLE_KEY || process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY);

async function main() {
  const { data, error } = await supabase.from('player_scores').select('replaced_player_id').limit(1);
  if (error) {
    console.error("Error fetching replaced_player_id:", error);
  } else {
    console.log("Success! Column exists. Sample data:", data);
  }
}
main();
